backend/tests/models/test_time.py

```diff
@@ -0,0 +1,93 @@
+"""Test APITime class."""
+from backend.lib.utils.datatypes import Point
+from backend.models import time
+
+import mock
+
+import pytest
+
+import requests
+
+NEWPORT_RI = Point(**{'longitude': -71.312796, 'latitude': 41.49008})
+CLEVELAND_OH = Point(**{'longitude': -81.695391, 'latitude': 41.499498})
+
+
+class MockResponse:
+    """Mock HTTP responses."""
+
+    def __init__(self, json_data, status_code):
+        self.json_data = json_data
+        self.status_code = status_code
+
+    def json(self):
+        return self.json_data
+
+    def raise_for_status(self):
+        if self.status_code != 200:
+            raise requests.exceptions.HTTPError(
+                '{}'.format(self.status_code),
+                response=self
+            )
+
+
+@mock.patch('requests.get')
+def test_get_matrix_http(mock_get):
```

Comment:
This is almost identical to the test in `test_mapbox.py` -- I think it's okay to just have one of them if the coverage isn't negatively affected.